test/utils/TestUtils.hs

```diff
@@ -159,9 +159,6 @@ stackYaml =
   "stack-8.0.2.yaml"
 #endif
 
-logFilePath :: String
-logFilePath = "functional-hie-" ++ stackYaml ++ ".log"
```

Comment:
This function is required by some tests. See ci-build

Reply:
A grep over the entire source doesn't show any other users of this definition. I don't know what you mean?

Reply:
Ugh, nvm. I'm an idiot :)